site/docs/configuration/expected-outputs/guardrail.md

```diff
@@ -0,0 +1,70 @@
+---
+sidebar_position: 101
+sidebar_label: Guardrails
+---
+
+# Guardrails
+
+Use the `guardrails` assert type to ensure that LLM outputs pass safety checks based on the provider's built-in guardrails.
+
+This assertion checks if the provider's response includes guardrails information and verifies that the content hasn't been flagged for safety concerns.
+
+## Basic Usage
+
+Here's a basic example of using the guardrail assertion:
+
+```yaml
+tests:
+  - vars:
+      prompt: 'Your test prompt'
+    assert:
+      - type: guardrails
+```
+
+You can also set it as a default test assertion:
+
+```yaml
+defaultTest:
+  assert:
+    - type: guardrails
+```
+
+## Redteaming Configuration
```

Comment:
Explain the normal pass fail logic before explaining the redteam customized pass fail logic.